src/main/webapp/app/exercises/text/manage/text-exercise-import.component.html

```diff
@@ -33,7 +33,7 @@ <h4 class="modal-title" jhiTranslate="artemisApp.textExercise.home.importLabel">
             <tbody>
                 <tr *ngFor="let exercise of content.resultsOnPage; trackBy: trackId" class="flex-row">
                     <td class="col-1">
-                        <span>{{ exercise.id }}</span>
+                        <span [class.ngb-highlight]="'' + exercise.id === searchTerm"> {{ exercise.id }}</span>
```

Comment:
As you have noticed during the development, you need to copy this value 5 times. I suggest to generalize all these components in a future PR.

And additionally, `[class.ngb-highlight]="exercise.id === searchTerm">` should be enough or?

Reply:
Exactly, we also have an issue regarding this https://github.com/ls1intum/Artemis/issues/5017 (which I did not have time to work on, yet).
With the changes here and in #5484, we should already keep in mind (or work towards) abstracting these components to reduce the amount of code duplication.

Reply:
> As you have noticed during the development, you need to copy this value 5 times. I suggest to generalize all these components in a future PR.
> 
> And additionally, `[class.ngb-highlight]="exercise.id == searchTerm">` should be enough or?

```
Error: src/main/webapp/app/exercises/quiz/manage/quiz-exercise-import.component.html:36:54 - error TS2367: This condition will always return 'false' since the types 'number | undefined' and 'string' have no overlap.

36                         <span [class.ngb-highlight]="exercise.id == searchTerm"> {{ exercise.id }}</span>
                                                        ~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Maybe cast `number | undefined` to string?